c2corg_ui/templates/helpers/view.html

```diff
@@ -247,6 +247,29 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
     </section>
 </%def>
 
+<%def name="show_associated_reports(document, type='reports')">\
```

Comment:
Do we really need this ``type`` argument? Isn't it always set to ``reports``? Then you can remove this argument and replace it by ``reports``.